sdks/swift/Mentat/Mentat/Extensions/UnsafeMutablePointerResult+Unwrap.swift

```diff
@@ -11,37 +11,43 @@
 import Foundation
 import MentatStore
 
-public extension Result {
+extension UnsafeMutablePointer where Pointee == Result {
     /**
-     Force unwraps a result.
-     Expects there to be a value attached and throws an error is there is not.
-
+     Unwraps an optional result, yielding either a successful value or a nil.
+     
+     Frees the Result afterwards, but not the object held in the `ok` property (which is what you want)
```

Comment:
How about: "... in the `ok` property.  Usually, this is what you want: in the success case, you extract the `ok` property and don't want it to vanish beneath you, but you do want the container to be released."